spec/swagger_helper.rb

```diff
@@ -54,6 +54,12 @@
           ApplicationResponse: APPLICATION_RESPONSE[version],
           ApplicationsResponse: APPLICATIONS_RESPONSE[version],
           Application: APPLICATION[version],
+          ApplicationAcceptRequest: APPLICATION_ACCEPT_REQUEST,
```

Comment:
Is it worth just nesting these into a single file (one for accept, one for reject and one for funded place)? I think having them split out when they aren't being shared across other schemas makes it harder to digest